storage-proofs/Cargo.toml

```diff
@@ -14,10 +14,14 @@ storage-proofs-post = { path = "./post", version = "4.0.1" }
 storage-proofs-porep = { path = "./porep", version = "4.0.1" }
 
 [features]
-default = ["gpu"]
+default = []
```

Comment:
I guess making it possible to disable the gpu feature makes sense. Though I think it should still be on by default.

Reply:
enable default gpu feature crates: `fil-proofs-tooling`、[filecoin-ffi](https://github.com/filecoin-project/filecoin-ffi/pull/125) 、[rust-filecoin-proofs-api](https://github.com/filecoin-project/rust-filecoin-proofs-api/pull/36)

other crates (`filecoin-proofs`、`storage-proofs`、`storage-proofs-core`、`storage-proofs-porep`、`storage-proofs-post`) are disabled default gpu feature.

is this ok ?

Reply:
@vmx BTW,  the file ` storage-proofs/porep/src/stacked/vanilla/proof.rs` has updated, now the code of SDR Precommit2 code can work in both cases (enable GPU or disable).

Reply:
@LuoZijun Thanks for those changes. I think that `gpu` should be enabled by default. In case that breaks your use case, we should talk about that in more detail.

Before you make any more changes, I'd like to my team first, if we want to have non-GPU support in rust-fil-proofs.